demo-front/src/app/components/main-page/main-page.component.html

```diff
@@ -1,18 +1,40 @@
-<table class="table table-bordered shadows" *ngIf="authService.isUserLoggedIn()">
-  <thead>
-  <tr>
-    <th scope="col">#</th>
-    <th scope="col">Name</th>
-    <th scope="col">Password</th>
-    <th scope="col">Phone</th>
-  </tr>
-  </thead>
-  <tbody>
-  <tr *ngFor="let user of users">
-    <th>{{user.id}}</th>
-    <td>{{user.username}}</td>
-    <td>{{user.password}}</td>
-    <td>{{user.phone}}</td>
-  </tr>
-  </tbody>
-</table>
+<!--<table class="table table-bordered shadows" *ngIf="authService.isUserLoggedIn()">-->
+<!--  <thead>-->
+<!--  <tr>-->
+<!--    <th scope="col">#</th>-->
+<!--    <th scope="col">Name</th>-->
+<!--    <th scope="col">Password</th>-->
+<!--    <th scope="col">Phone</th>-->
+<!--  </tr>-->
+<!--  </thead>-->
+<!--  <tbody>-->
+<!--  <tr *ngFor="let user of users">-->
+<!--    <th>{{user.id}}</th>-->
+<!--    <td>{{user.username}}</td>-->
+<!--    <td>{{user.password}}</td>-->
+<!--    <td>{{user.phone}}</td>-->
+<!--  </tr>-->
+<!--  </tbody>-->
+<!--</table>-->
+<div style="align-content: center;margin: 50px; justify-content: center" *ngIf="authService.isUserLoggedIn()">
```

Comment:
Add class attribute and apply css for this class, please. Here and in few other places.